### PR TITLE
Fix/#236 Uncompilable code generation caused by invalid variable name

### PR DESCRIPTION
**Pull request by jiachen1120**
Related issue: https://github.com/networknt/light-codegen/issues/236

Related RFC: https://github.com/networknt/light-rfcs/blob/master/light-codegen/0005-fix-uncompilable-generation-caused-by-naming.md

Related Jira ticket: API-73

Look forward to your reviews!

### CODE REVIEW

**Comment by NicholasAzar** [comment]
@jiachen1120 could we use the `SourceVersion.isName(..)` method to simplify this maybe? https://docs.oracle.com/javase/8/docs/api/index.html?javax/lang/model/SourceVersion.html

**Comment by NicholasAzar** [comment]
@jiachen1120 could you resolve the conflicts on this?

File: light-rest-4j/src/main/java/com/networknt/codegen/rest/SwaggerGenerator.java [hunk 1]
```diff
@@ -31,6 +31,14 @@
     //static ObjectMapper mapper = new ObjectMapper();
 
     private Map<String, String> typeMapping = new HashMap<>();
+
+    private static List<String> javaKeyWordList = new ArrayList<>(Arrays.asList("abstract", "continue", "for",	"new",
+            "switch", "assert", "default", "goto", "package", "synchronized", "boolean", "do", "if", "private", "this",
```
Comment (NicholasAzar) [inline light-rest-4j/src/main/java/com/networknt/codegen/rest/SwaggerGenerator.java]: Seems like you have different implementations of detecting java keywords between openapi and swagger now

Comment (jiachen1120) [reply light-rest-4j/src/main/java/com/networknt/codegen/rest/SwaggerGenerator.java]: @NicholasAzar I forgot to update the swagger one, I will modify the swagger one to sync with openapi

**Pull request merged**